### DISCUSSION TITLE
JetBrains Mono not in schema.json?

### DISCUSSION THREAD

**Discussion by broktoram** [squidfunk/mkdocs-material#5384] [General]
I'm using the schema.json file to validate mkdocs.yml. 

If I use JetBrains Mono as the code font, I'm getting a "Value is not accepted" message and I can't seem to find the font in the list of valid values even though it's part of Google Fonts. Am I missing something?

**Comment by squidfunk** [comment]
We're using [`google-fonts-complete`](https://github.com/jonathantneal/google-fonts-complete) to generate the list, so there would need to be a new version of that package that includes the font. Maybe you can open an issue on the repository and ask for a new release?

**Reply by broktoram** [reply]
Thanks, I've created a new issue there but it looks like that repo hasn't been updated in a long time...
